### PR TITLE
Initial implementation of the passwordreset page

### LINKED ISSUE #24: External account reset is not implemented
See title, under login box there is no implementation for external account reset. Related to #23

### PR DESCRIPTION
**Pull request by noelvandevrie**
This includes changes to:

- router/index.ts
- PasswordReset.vue
- LoginView.vue
- en.json
- nl.json
- main.ts

Fixes: #24

### CODE REVIEW

File: src/views/PasswordResetView.vue
Comment (CodeNamedRobin) [inline src/views/PasswordResetView.vue:1]: There are some console logs left in there.

File: src/views/PasswordResetView.vue [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+});
+
+const passwordForm = useForm({
+  validationSchema: passwordSchema,
+});
+
+const passwordResetMode = ref(0);
+const email = emailForm.defineComponentBinds('email');
+const password = passwordForm.defineComponentBinds('password');
+const password2 = passwordForm.defineComponentBinds('password2');
```
Comment (CodeNamedRobin) [inline src/views/PasswordResetView.vue]: Might be nicer and more informative to refactor password2 to passwordConfirm?

File: src/views/PasswordResetView.vue [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+
+const backToLogin = () => {
+  router.push({ name: 'login' });
+};
+
+</script>
+
+<style scoped lang="scss">
+//TODO Cleanup and fix
```
Comment (CodeNamedRobin) [inline src/views/PasswordResetView.vue]: Please add a link to #14.

**Review by CodeNamedRobin — CHANGES_REQUESTED** [review]
Some small things.

File: src/views/PasswordResetView.vue
Comment (noelvandevrie) [reply src/views/PasswordResetView.vue:1]: These are now gone in the latest commit to this branch.

File: src/views/PasswordResetView.vue (hunk 1 above)
Comment (noelvandevrie) [reply src/views/PasswordResetView.vue]: Updated in the latest commit to this branch.

File: src/views/PasswordResetView.vue (hunk 2 above)
Comment (noelvandevrie) [reply src/views/PasswordResetView.vue]: Added TODO's for #14 and #25 in PasswordResetView, also updated the description to include #24

**Review by CodeNamedRobin — APPROVED** [review]
LGTM!

**Pull request merged**